db/validate.js

```diff
@@ -0,0 +1,10 @@
+exports.courseDetails = (course) => {
+  var start_m = Date.parse(course.startdate)
+  var end_m = Date.parse(course.enddate)
+  var name_m = course.coursename.trim()
+  var currentDateArr = Date().split(' ')
+  var newCurrentDate = currentDateArr[3] + '/' + (new Date().getMonth() + 1) + '/' + currentDateArr[2]
+  var newCurrentDate_m = Date.parse(newCurrentDate)
+  if (name_m.length > 0 && end_m > start_m && start_m >= newCurrentDate_m) return course
```

Comment:
What does this logic do? Give it a name, extract a function, then consider naming the pieces. Prefer functions to variables.

The book "Refactoring" has great explanations of how and why to do method extraction.